token/token_context.go

```diff
@@ -1,6 +1,4 @@
-// Package token_context contains the code that extract token manager from the
-// context.
-package tokencontext
+package token
```

Comment:
make sure if there are any service using this currently then we need to adopt it with this change :+1:

Reply:
for now no one uses it directly, but it's also currently duplicated in other repos.